public_html/assets/js/nf-core-schema-builder.js

```diff
@@ -926,7 +927,104 @@ $(function () {
         // Update printed schema in page
         $('#json_schema').text(JSON.stringify(schema, null, 4));
     });
+    //
+    // multi-select modal
+    //
+    
+    $('#multi_select_modal').on('change', '.select_param', function(){
+        var num_selected = $('#multi_select_modal').find('.select_param:checked').length;
+        if(num_selected>0){
+            $('#multi_select_modal #move_params').removeClass("disabled");
+            if(num_selected === 1){
+                $('#multi_select_modal #move_params').html("Move 1 parameter");
+            } else {
+            $('#multi_select_modal #move_params').html("Move "+num_selected+" parameters");
+            }
+        } else{
+            $('#multi_select_modal #move_params').addClass("disabled");
+            $('#multi_select_modal #move_params').html("Move parameters");
+        }
+    })
+    $('#schema-builder').on('click', '.schema_group_move_params', function () {
+        // Get row
+        var row = $(this).closest('.schema_group');
+        var id = row.data('id');
+        launch_multi_select_modal(id);
+    });
+    function launch_multi_select_modal(id) {
+        // Reset button to initial state
+        $('#multi_select_modal #move_params').addClass("disabled");
+        $('#multi_select_modal #move_params').html("Move parameters");
+        // Build modal
+        $('#multi_select_modal .modal-header h4 span').html(id)
+        var params = '';
+        for (k in schema['properties']) {
+            // skip over Groups
+            if (schema['properties'][k].hasOwnProperty('properties')) {
+                continue
+            }
+            // Add to the preview
+            params += `
+                <tr>
+                    <td>
+                        <input type="checkbox" aria-label="Move this parameter" class="select_param" data-id=`+k+` id="group-move-`+k+`">
+                    </td>
+                    <td>
+                        <label for="group-move-`+k+`" class="text-monospace">`+k+`</label>
+                    </td>
+                    <td>
+                        <label for="group-move-`+k+`" class="small">`+ schema['properties'][k].description +`</label>
+                    </td>
+                </tr>
+                `
+        }
+        if (params===''){
+            params = '<div class="alert alert-info">No ungrouped parameters available.</div>';
+            $('#multi_select_modal .table').remove();
```

Comment:
`remove()` breaks this if the user moves any params back to the top level. It shows the "no ungrouped" message forever more...

Better to have the alert in the main HTML but hidden by default, then just toggle their visibilities..

Reply:
fixed in 36e9458